app/components/ui/tale-tab-metadata/component.js

```diff
@@ -0,0 +1,91 @@
+import Ember from 'ember';
+import { later } from '@ember/runloop';
+import { computed } from '@ember/object';
+import { not } from '@ember/object/computed';
+
+import $ from 'jquery';
+
+import config from '../../../config/environment';
+
+const taleStatus = Object.create({
+  NONE: -1,
+  READ: 0,
+  WRITE: 1,
+  ADMIN: 2,
+  SITE_ADMIN: 100
+});
+
+export default Ember.Component.extend({
+  apiHost: config.apiHost,
+  environments: [],
+  model: null,
+  init() {
+    this._super(...arguments);
+    
+    // Fetch images for user to select an environment
+    let component = this;
+    $.getJSON(this.get('apiHost') + '/api/v1/image/').then(function(images) {
+      component.set('environments', images);
+      component.selectDefaultImageId();
+    });
+  },
+  
+  didRender() {
+    // Enable the environment dropdown functionality
+    $('.ui.dropdown').dropdown();
+    this.selectDefaultImageId();
+  },
+  
+  selectDefaultImageId() {
+    // Select the current imageId by default
+    let selectedImageId = this.get("model").get("tale").get("imageId")
+    $('.ui.dropdown').dropdown('set selected', selectedImageId);
+  },
+  
+  canEditTale: computed('model.tale._accessLevel', function () {
+    return this.get('model') && this.get('model').get('tale') && this.get('model').get('tale').get('_accessLevel') >= taleStatus.WRITE;
+  }).readOnly(),
+  cannotEditTale: computed.not('canEditTale').readOnly(),
+  
+  actions: {
+    // TODO: Status messages are still missing
+    updateTale() {
+      let tale = this.get("model").get("tale");
+      let component = this;
+      //component.set("tale_creating", true);
+      console.log("Sending updated tale:", tale);
+      console.log("New image ID:", tale.imageId);
+      
+      let onSuccess = (item) => {
+      //component.set("tale_creating", false);
+      //component.set("tale_created", true);
+      
+      later((function () {
+        //component.set("tale_created", false);
+        // component.transitionToRoute('upload.view', item);
+      }), 10000);
+      console.log("Successfully saved tale (" + tale['id'] + "):", tale)
+      };
+      
+      let onFail = function (e) {
+      // deal with the failure here
+      //component.set("tale_creating", false);
+      //component.set("tale_not_created", true);
+      //component.set("error_msg", e.message);
+      
+      later((function () {
+        //component.set("tale_not_created", false);
+      }), 10000);
+      console.log("Error saving tale (" + tale['id'] + "):", tale)
```

Comment:
This may not be directly related to this PR, but I'll comment here for reference. We might want to notify the user that their Tale failed to save and to try again. If it's not new code we can probably put an issue in for it, if it's needed. @craig-willis what do you think?

Reply:
I can easily change the `console.log` to a classic `alert()` for the time being. There is apparently a warning modal that we may be able to reuse if we would rather have fancy styling on the error message

Reply:
I wasn't aware of `Ember.onerror`; it looks pretty handy. We could put a common error dialog into a service and inject it into the various components, but that sounds like a separate task.